### ISSUE TITLE
Compilation with RiVLib fails

### ISSUE THREAD

**Issue by sedot42** [PDAL/PDAL#2359]
Commit 83bf2dccaf1 breaks compilation with RiVLib for me. I don't understand the build process well enough to fix it. CMake warns that linked items are dropped.
```
WARNING: Target "rxptest" requests linking to directory "/home/stefan/Documents/Uni/Heidelberg/Masterarbeit/05_processing/01_preprocessing/tools/pdal_compilation/rivlib-2_5_7-x86_64-linux-gcc55/include".  Targets may link only to libraries.  CMake is dropping the item.
WARNING: Target "pdal_plugin_reader_rxp" requests linking to directory "/home/stefan/Documents/Uni/Heidelberg/Masterarbeit/05_processing/01_preprocessing/tools/pdal_compilation/rivlib-2_5_7-x86_64-linux-gcc55/include".  Targets may link only to libraries.  CMake is dropping the item.
```
This results in the fatal error ```plugins/rxp/io/RxpPointcloud.hpp:48:10: fatal error: riegl/scanlib.hpp: No such file or directory```

@abellgithub Could you have a look, please?

**Comment by gadomski** [comment]
@sedot42 I have a potential fix in #2360, if you're able to pull that branch and confirm the fix that'd be helpful. You'll need to have `BUILD_RIVLIB_TESTS=ON` as the breakage occurs while building the rxp test.

**Comment by sedot42** [comment]
@gadomski It builds. Nice!
